src/modules/Search/Search.d.ts

```diff
@@ -30,7 +30,7 @@ export interface SearchProps {
   noResultsDescription?: string;
```

Comment:
@layershifter Should I add specific test for case when noResultsDescription isn't a string?